### ISSUE TITLE
Resource instance list cards display values on a single line

### ISSUE THREAD

**Issue by rlwilliamss** [archesproject/arches#8228]
**Describe the bug**

For cards using a resource instance list, when multiple values are added, all values are displayed in the card tree on one line making it difficult to read as you need to drag the tree to view all the values.

**To Reproduce**

1. Open a Heritage Asset record with multiple values added for any of the cards using a resource instance list e.g., Associated Digital Files.

2. You need to drag the card tree to view all the values in the card:

**Screenshots**

![image](https://user-images.githubusercontent.com/78143228/157628323-d5ae8323-b1d2-41f7-9189-0d143ffbc232.png)

**Expected behavior**
To make it easier to read, it would be nice to have each value displayed on a new line if possible.

**Your Arches Information**
*   Version used: 6.1.x
*   Operating System and version (desktop or mobile): Desktop Windows 10
*   Browser Name and version: Chrome 
*   Link to your Arches Install (optional):

**Ticket Background**
*   Found by: @aj-he

**Comment by chiatt** [comment]
Maybe we should avoid showing all the related instance names in the tree because the complete list is always available in the card on the right (rendered as a report or a form). Some nodes could have a lot of related resources making the tree really long and difficult to read. Instead of listing them, maybe just summarizing them would be safer? Perhaps something like: 
`Digital File(s): 5 related instances`. What do you think @rlwilliamss & @dwuthrich?

**Comment by rlwilliamss** [comment]
@chiatt Yes I like this idea, I think it would be easier for users to see a summary in the tree.
